index.html

```diff
@@ -309,11 +309,30 @@
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] contains [=verification
-relationships=] that explicitly permit the use of certain [=verification
-methods=] for specific purposes.
+A [=controller document=] contains cryptographic material and service
+endpoints for the purposes of verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+      </p>
+      <p>
+[=Controller documents=] enable the verification of proofs created
+by the controller of an identifier. They provide [=verification methods=]
+that express public cryptographic material, such as public keys, for 
+verifying proofs created by the controller of the 
+identifier for specific [=verification relationships=], such as
+authentication, attestation, encryption, and capability invocation
+and delegation.
```

Comment:
Introducing the concept of "verification relationships" too early in the document feels like it might overload readers with too much specialized language. While it is true that the property determines the relationship between the subject and the verification method (that is, it really is a "verification relationship"), focusing on that level of detail misses the larger point that it's for establishing the purpose of the verification methods existence. I suggest we use "purpose" instead of "verification relationship" to convey the higher level concepts in the introduction. We can get more pedantic later in the specification.

```suggestion
that express public cryptographic material, such as public keys, for 
verifying proofs created by the controller of the 
identifier for specific purposes, such as
authentication, attestation, encryption, and capability invocation
and delegation.
```